src/assessments/seo/largestKeywordDistanceAssessment.js

```diff
@@ -0,0 +1,133 @@
+const AssessmentResult = require( "../../values/AssessmentResult.js" );
+const Assessment = require( "../../assessment.js" );
+const merge = require( "lodash/merge" );
+const countWords = require( "../../stringProcessing/countWords.js" );
+const matchWords = require( "../../stringProcessing/matchTextWithWord.js" );
+const Mark = require( "../../values/Mark.js" );
+const marker = require( "../../markers/addMark.js" );
+const inRangeStartEndInclusive = require( "../../helpers/inRange.js" ).inRangeStartEndInclusive;
+
+/**
+ * Returns a score based on the largest percentage of text in
+ * which no keyword occurs.
+ */
+class largestKeywordDistanceAssessment extends Assessment {
+	/**
+	 * Sets the identifier and the config.
+	 *
+	 * @param {Object} config The configuration to use.
+	 *
+	 * @returns {void}
+	 */
+	constructor( config = {} ) {
+		super();
+
+		const defaultConfig = {
+			overRecommendedMaximumKeywordDistance: 40,
+			recommendedMaximumKeywordDistance: 30,
```

Comment:
I am debating with myself if lowing down these boundaries would be a smart move. On the one hand, all well-written texts I tested (a.k.a. Yoast Blog) pass this test with flying colors. On the other hand, with non-working morphology these boundaries might be a little too low for many amateur writers. Could you please give these boundaries a little more test?

Reply:
I've raised the boundaries by 10%. So now you only get a bad score if half of your text doesn't contain the keyword, and an okay score if 40% of your text doesn't contain your keyword.